Mesh_3/include/CGAL/Mesh_3/Mesher_3.h

```diff
@@ -719,6 +720,20 @@ initialize()
   else
 #endif // CGAL_LINKED_WITH_TBB
   {
+
+    if (r_c3t3_.number_of_facets() == 0)
+    {
+      CGAL::warning_fail("c3t3.number_of_facets() == 0",
+        "CGAL/Mesh_3/Mesher_3.h",
```

Comment:
@CGAL/geometryfactory Here Jane and I have decided to call `CGAL::warning_fail` directly, because we wanted a warning that is not disable in Release mode. Is that a good idea? The advantage of `CGAL::warning_fail` over a plain display to `std::cerr` is that the behavior can be [tweaked by the user/developer, using behaviors and handlers](https://doc.cgal.org/latest/STL_Extension/group__PkgSTLExtensionAssertions.html).

Should it be a macro, so that it can be disabled anyway with a dedicated macro definition?